mw-webapp/src/component/progressBar/ProgressBar.tsx

```diff
@@ -0,0 +1,34 @@
+import styles from "src/component/progressBar/ProgressBar.module.scss";
+ type ProgressBarProps = {/**
+                           * ProgressBar prop type
+                           */
+   progress: Array<boolean>;}
```

Comment:
what if it is empty? This behavior not specified

Reply:
Done